frontend/cms-angular/src/app/components/admin/admin.component.html

```diff
@@ -0,0 +1,31 @@
+
+<body>
+  <div id="wrapper">
+     <!-- Sidebar -->
+     <div id="sidebar-wrapper">
+        <ul class="sidebar-nav nav-pills nav-stacked" id="menu">
+           <li class="active">
+              <a href="admin/login"> <span class="fa-stack fa-lg pull-left"></span> Dashboard</a>
+           </li>
+           <li>
+              <a href="/admin/register"><span class="fa-stack fa-lg pull-left"></span>Pages</a>
+           </li>
+           <li>
+              <a href="#"><span class="fa-stack fa-lg pull-left"></span>Add Article</a>
+           </li>
+        </ul>
+     </div>
+
+     <!-- /#sidebar-wrapper -->
+     <!-- Page Content -->
+     <div id="page-content-wrapper">
+        <div class="container-fluid xyz">
+          <router-outlet></router-outlet>        
+        </div>
+     </div>
+     <!-- /#page-content-wrapper -->
+  </div>
+  <!-- /#wrapper -->
+  <!-- jQuery -->
```

Comment:
nie wiem skąd to kopiowałeś ale fajnie byłoby te wszystkie komentarze sprzątnąć bo są zbędne